executor/simple.go

```diff
@@ -659,6 +653,12 @@ func (e *SimpleExec) executeCreateUser(ctx context.Context, s *ast.CreateUserStm
 		if s.IsCreateRole {
 			user = fmt.Sprintf(`('%s', '%s', '%s', 'Y')`, spec.User.Hostname, spec.User.Username, pwd)
 		}
+		if exists {
+			if !s.IfNotExists {
+				return infoschema.ErrUserAlreadyExists.GenWithStackByArgs(user)
```

Comment:
```suggestion
				return infoschema.ErrUserAlreadyExists.GenWithStackByArgs(fmt.Sprintf(`'%s'@'%s'`), spec.User.Hostname, spec.User.Username)
```
How about changing the message like this?